users/arxiv/users/auth/scopes.py

```diff
@@ -77,24 +77,55 @@
 """Authorizes releasing an upload workspace."""
 
 DELETE_UPLOAD_WORKSPACE = Scope(Scope.domains.UPLOAD, 'delete_workspace')
+"""Can delete an entire workspace in the file management service."""
+
 DELETE_UPLOAD_FILE = Scope(Scope.domains.UPLOAD, Scope.actions.DELETE)
+"""Can delete files from a file management upload workspace."""
 
 READ_UPLOAD_LOGS = Scope(Scope.domains.UPLOAD, 'read_logs')
+"""Can read logs for a file management upload workspace."""
+
 READ_UPLOAD_SERVICE_LOGS = Scope(Scope.domains.UPLOAD, 'read_service_logs')
+"""Can read service logs in the file management service."""
 
 READ_COMPILE = Scope(Scope.domains.COMPILE, Scope.actions.READ)
+"""Can read products generated by the compilation service."""
```

Comment:
Minor. Do we expect the compilation service to compile anything besides documents?
```suggestion
"""Can read documents generated by the compilation service."""
```

Reply:
Good catch; nope!